libsql_client/hrana/conn.py

```diff
@@ -289,6 +336,30 @@ def get_result(response: proto.Response) -> proto.StmtResult:
             return cast(proto.ExecuteResp, response)["result"]
         return _map_future(response_fut, get_result)
 
+    def sequence(self, stmt: Union[str, int]) -> asyncio.Future[None]:
```

Comment:
Following our discussion with Doug, let's change this to `execute_multiple`.

Reply:
this is the internal one, not the client. I'd keep it like the protocol, when someone adds the high level, then go with `execute_multiple`